Makefile

```diff
@@ -21,6 +21,8 @@ PROJECT_VERSION					 ?= project-version
 
 INPUT_REPORT_FILE_PATH			 ?= input-report
 
```

Comment:
Can you also define the `SPREADSHEET_URL` after the `SOURCE_URL` variable? It's just for the sake of defining the variable. 

`SPREADSHEET_URL   ?= http://<<please-google-spreadsheet-url>>`

Reply:
we replaced this parameter with INPUT_REPORT_FILE_PATH
but i have set the value you meantioned for readability